### PR TITLE
Use ansible-runner in APB Base

### PR DESCRIPTION
**Pull request by djzager**
This PR updates apb-base to be based off `ansible/ansible-runner`.

### CODE REVIEW

File: Dockerfile-latest [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM centos:7
+FROM ansible/ansible-runner:latest
```
Comment (jmontleon) [inline Dockerfile-latest]: Since they don't have an rpm based container upstream maybe we can install ansible-runner from EPEL instead.

Comment (djzager) [reply Dockerfile-latest]: I don't get the impression they are keeping the RPM up to date. I feel like the best way to stay up-to-date is with the container image but I can make the change if that's best.

```
bash-4.2# yum -y install python-ansible-runner
bash-4.2# rpm -qa | grep 'ansible-runner'
python2-ansible-runner-1.0.1-1.el7.noarch
```

**Comment by djzager** [comment]
[Integrate Ansible Runner with apb-base](https://trello.com/c/ILPjFVHE/798-integrate-ansible-runner-with-apb-base)

**Comment by djzager** [comment]
Tested with and without RPM install

**Comment by jmontleon** [comment]
ACK!

**Pull request merged**